cudamapper/samples/sample_cudamapper.cpp

```diff
@@ -126,7 +127,16 @@ void process_batch(std::vector<IndexDescriptor>& query_index_descriptors,
                 // print overlaps
                 if (print)
                 {
-                    print_paf(overlaps, cigars, *query_parser, *target_parser, KMER_SIZE, print_mutex);
+                    if (!sam_format)
```

Comment:
if `sam_format` is selected, but it's not built with sam support, looks like it might silently not print anything?

Reply:
fixed by adding ifdef guards, printing a warning, and exiting on use without building with sam/bam support

Reply:
Yep that's good!